modules/packages/LockFreeStack.chpl

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module LockFreeStack {
```

Comment:
If this is intended to be a user-facing data structure, it needs top-level chpldoc comments and API documentation within.

If it's just being added as a test for EBR, just adding it to `test/` somewhere is more appropriate.

Reply:
@dgarvit You should apply @mppf's feedback that he left on `gitter`. We need more documentation for the lock free modules since we plan to actually expose them to the user.